src/components/Checkbox/Checkbox.js

```diff
@@ -16,51 +19,67 @@ const Checkbox = ({
   isMyAccount,
   className,
   disabled,
-  isRadioButton
+  isRadioButton,
+  termsUrl,
+  isPayPal
 }) => {
   const [isChecked, setIsChecked] = useState(checked);
+  const { t } = useTranslation();
 
   useEffect(() => {
     setIsChecked(checked);
   }, [checked]);
 
   return (
-    <CheckboxStyled
-      onClick={e => {
-        e.stopPropagation();
-        onClickFn(e, disabled, setIsChecked);
-      }}
-      role="checkbox"
-      tabIndex="-1"
-      aria-checked={isChecked}
-      checked={isChecked}
-      aria-label={children}
-      className={className}
-      disabled={disabled}
-    >
-      <CheckFrameStyled
-        error={error && required && !isChecked}
-        tabIndex="0"
-        onKeyDown={e => (e.keyCode === 32 ? onClickFn() : null)}
-        isMyAccount={isMyAccount}
-        isRadioButton={isRadioButton}
-        checked={isChecked}
-      >
-        {isChecked && (
-          <CheckMarkStyled
-            data-testid="checkmark"
-            isMyAccount={isMyAccount}
-            isRadioButton={isRadioButton}
-          />
-        )}
-      </CheckFrameStyled>
-      <ConsentDefinitionStyled
-        dangerouslySetInnerHTML={{
-          __html: `${children}${required && isMyAccount ? '*' : ''}`
+    <>
+      <CheckboxStyled
+        onClick={e => {
+          e.stopPropagation();
+          onClickFn(e, disabled, setIsChecked);
         }}
+        role="checkbox"
+        tabIndex="-1"
+        aria-checked={isChecked}
         checked={isChecked}
-      />
-    </CheckboxStyled>
+        aria-label={children}
+        className={className}
+        disabled={disabled}
+      >
+        <CheckFrameStyled
+          error={error && required && !isChecked}
+          tabIndex="0"
+          onKeyDown={e => (e.keyCode === 32 ? onClickFn() : null)}
```

Comment:
Maybe we could make variable above with this 32 number
`const spaceKeyCode === 32`